Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/TabbedPage.cs

```diff
@@ -150,52 +151,71 @@ public static int GetMaxItemCount(this IPlatformElementConfiguration<Android, Fo
 			return GetMaxItemCount(config.Element);
 		}
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		[Obsolete("This property is obsolete as of version 3.5. Please use TabbedPage.UnselectedTabColor instead.")]
 		public static readonly BindableProperty BarItemColorProperty =
 			BindableProperty.Create("BarItemColor", typeof(Color),
 			typeof(TabbedPage), Color.Default);
```

Comment:
I would actually add an `propertyChanged: OnChange` here, so that this property is responsible for setting the real property. This way they are in sync and still works if it is set via `SetValue`. All the existing code in the methods here can remain unchanged, but unused by us.